deploy/downstream-prep.sh

```diff
@@ -66,3 +77,18 @@ sed -i "/MIG_UI_TAG/,/^ *[^:]*:/s/value: release-1.0/value: ${V1_0_IMG_MAP[ui_sh
 sed -i "/VELERO_PLUGIN_TAG/,/^ *[^:]*:/s/value: release-1.0/value: ${V1_0_IMG_MAP[plugin_sha]}/"              deploy/olm-catalog/mig-operator/v1.0.1/mig-operator.v1.0.1.clusterserviceversion.yaml
 sed -i "/VELERO_TAG/,/^ *[^:]*:/s/value: fusor-1.1/value: ${V1_0_IMG_MAP[velero_sha]}/"                       deploy/olm-catalog/mig-operator/v1.0.1/mig-operator.v1.0.1.clusterserviceversion.yaml
 sed -i "/VELERO_RESTIC_RESTORE_HELPER_TAG/,/^ *[^:]*:/s/value: fusor-1.1/value: ${V1_0_IMG_MAP[helper_sha]}/" deploy/olm-catalog/mig-operator/v1.0.1/mig-operator.v1.0.1.clusterserviceversion.yaml
+
+# Make 1.0.1 Downstream non-OLM changes
```

Comment:
It looks like there was a commit in the release-1.0 branch to make these changes for v1.0.0 that never made it into master.

Doing it this way would mean also needing to run the downstream-prep.sh on the operator container, but I think it accomplishes two things:
- Pins images to a sha like the CSV is doing
- Makes it clearer (for us) to see where these changes happened. I was initially confused not seeing what I expected in upstream master and later relalized the change was only in release.